packages/react-native-gesture-handler/src/v3/hooks/callbacks/GestureHandlerTouchEventWorkletHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { NativeSyntheticEvent } from 'react-native';
+import { CallbackHandlers, TouchEvent } from '../../types';
+import {
+  isEventForHandlerWithTag,
+  isNativeEvent,
+  runCallback,
+  touchEventTypeToCallbackType,
+} from '../utils';
+import { TouchEventType } from '../../../TouchEventType';
+import { GestureTouchEvent } from '../../../handlers/gestureHandlerCommon';
+
+export function getGestureHandlerTouchEventWorkletHandler(
```

Comment:
Same

Reply:
Also done in [feaa167](https://github.com/software-mansion/react-native-gesture-handler/pull/3682/commits/feaa1672580df65add84469c5f3e74d8730bc43d)